### DISCUSSION TITLE
Hot to configure IPBan to block immediately shell attack?

### DISCUSSION THREAD

**Discussion by marcolungo** [DigitalRuby/IPBan#203] [Q&A]
Hello, everyone,
I am looking for ways to block via IPBan shell attacks that are becoming more and more frequent. I am not very experienced, I have a Windows Server 2019 installed, and maybe it can be solved via Windows Firewall but I haven't found anything. I saw that in IPBan there is the possibility to create rules to block something, however I am not clear on how it works, also because I do not speak English well and I use online translators to understand. 
I get by with regex, and something as a log parser I managed to do, however it is not enough.
Is there, then, a way to immediately block shell attacks using IPBan or something else?
Thanks in advance to those who will answer me

Marco

Translated with www.DeepL.com/Translator (free version)

**Comment by jjxtra** [comment]
Can you post your event viewer xml or log file example?
